docs/conf.py

```diff
@@ -90,8 +87,9 @@
     # Building on a local Read the Docs instance
     hoverxref_api_host = 'http://community.dev.readthedocs.io'
 
-# TODO: remove me when EmbedAPIv3 gets deployed in production
-hoverxref_api_host = 'https://readthedocs.ngrok.io'
+if os.environ.get('NGROK_READTHEDOCS') == 'True':
+    # Building on a local Read the Docs instance using NGROK for HTTPS
+    hoverxref_api_host = 'https://readthedocs.ngrok.io'
```

Comment:
Shouldn't this value get pulled from the environment, so other people can use it if needed in dev? eg `NGROK_HOVERXREF= https://foo.ngrok.io`.

Reply:
Haha! Yes! I don't know why I didn't do it like you are suggesting originally 🙃  It should probably just use `hoverxref_api_host` all in uppercase.

Reply:
I suppose that I hardcoded it here so I don't have to remember the different URL options 😄